### PR TITLE
Api token get content api routes

### PR DESCRIPTION
**Pull request by Bassel17**
### What does it do?

Adds an admin route to get content api routes

### How to test it?

- Run a Strapi app and try to get the routes using `admin/content-api/routes`
it should return a structure similar to this 
```js
{
data:{
    'api::address': [
      {
        method: 'GET',
        path: '/api/addresses',
        handler: 'api::address.address.find',
        config: {
          auth: false,
        },
        info: {
          apiName: 'address',
          type: 'content-api',
        },
      },
      {
        method: 'GET',
        path: '/api/addresses/:id',
        handler: 'api::address.address.findOne',
        config: {
          auth: {
            scope: ['api::address.address.findOne'],
          },
        },
        info: {
          apiName: 'address',
          type: 'content-api',
        },
      },
    ],
   }
 }
  ```

### CODE REVIEW

File: packages/core/admin/server/services/content-api.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const _ = require('lodash');
```
Comment (Convly) [inline packages/core/admin/server/services/content-api.js:3]: I think this should live in the content API container next to the permissions property. IMO it's a content-api-related service, not necessarily an admin one

**Review by Convly — CHANGES_REQUESTED** [review]
Not sure to understand the value of adding the info property to the response objects.

- Method, path, and handler are needed to display info and map the route to the action
- config can be interesting to display information on whether you need authorizations to access the route or not
- info... I don't see the point tbh

File: packages/core/admin/server/services/content-api.js (hunk 1 above)
Comment (innerdvations) [reply packages/core/admin/server/services/content-api.js:3]: Yeah, that makes sense to me too.

Comment (Bassel17) [reply packages/core/admin/server/services/content-api.js]: info would be useful, we want the name, we can get it through different means in the frontend but I don't see why dele would do that, and it's already available (I am not doing anything extra to have this property) I don't see the point of deliberately removing it.

Comment (Convly) [reply packages/core/admin/server/services/content-api.js]: If it makes sense because it's needed in the frontend then why not :slightly_smiling_face:

**Review by Convly — CHANGES_REQUESTED** [review]
LGTM, but it looks like you've some conflicts :eyes:

**Review by Convly — APPROVED** [review]
LGTM waiting for the tests to pass

**Pull request merged**